views/js/qtiCreator/helper/xincludeLoader.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA;
+ */
+
+define(
+    ['jquery', 'lodash',],
+    function ($, _) {
+        'use strict';
+
+        const xincludeLoader = {
+            name: 'xincludeLoader',
+            load: function load(xincludeElement, baseUrl) {
+                const self = this;
+                const href = xincludeElement.attr('href');
+
+                // Return a new Promise
+                return new Promise((resolve, reject) => {
+                    if (href && baseUrl) {
+                        const fileUrl = `text!${baseUrl}${href}`;
+                        // reset the previous definition of the XML, to receive updated passage
+                        require.undef(fileUrl);
+                        // require xml
+                        require([fileUrl], function (stimulusXml) {
+                            const data = self.parseXmlToDom(stimulusXml);
+                            resolve({ xinclude: xincludeElement, data }); // Resolve with an object containing xinclude and data
+                        }, function () {
+                            // In case the file does not exist, reject the promise
+                            reject(new Error('File not found'));
+                        });
+                    } else {
+                        reject(new Error('href or baseUrl is missing'));
```

Comment:
I'm getting this error when try to get back to the item with Stimulus in the Authoring